### ISSUE TITLE
why weight_decay = 0?

### ISSUE THREAD

**Issue by accountcwd** [LiMeng95/MultiPoseNet.pytorch#29]
不知道我的理解有没有错误：weight_decay是给正则项的一个参数，如果为0，那就是不使用正则项。

为什么设置为0呢？是因为数据量足够不用担心过拟合的问题吗？

**Comment by LiMeng95** [comment]
是的，一般来说weight_dacay起到L2正则的作用，这个超参数也是可以调整的，之前我实验的时候没有太多关心这个超参设置。设置为0的原因是，在Adam中，weight_decay并不等价于L2正则，反而会与adam优化器存在一些冲突，可以参考：[L2正则=Weight Decay？并不是这样](https://zhuanlan.zhihu.com/p/40814046)

如果您调整weight_decay得到更好的结果的话，欢迎提PR。

**Comment by accountcwd** [comment]
> 
> 
> 是的，一般来说weight_dacay起到L2正则的作用，这个超参数也是可以调整的，之前我实验的时候没有太多关心这个超参设置。设置为0的原因是，在Adam中，weight_decay并不等价于L2正则，反而会与adam优化器存在一些冲突，可以参考：[L2正则=Weight Decay？并不是这样](https://zhuanlan.zhihu.com/p/40814046)
> 
> 如果您调整weight_decay得到更好的结果的话，欢迎提PR。

感谢您的详细回复!
